src/praisonai-agents/praisonaiagents/agent/execution_mixin.py

```
@@ -570,39 +570,62 @@ async def collect():
+                    bridge_queue = queue.Queue()
+
+                    def run_in_thread_noloop():
+                        new_loop = asyncio.new_event_loop()
+                        asyncio.set_event_loop(new_loop)
+                        try:
+                            async def pump():
+                                try:
+                                    async for item in self.backend.stream(prompt, **kwargs):
+                                        bridge_queue.put(('item', item))
+                                    bridge_queue.put(('done', None))
+                                except Exception as e:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)